cmd/otelcol/config/collector/gateway_config.yaml

```diff
@@ -102,7 +102,10 @@ exporters:
   # Debug
   #logging:
     #loglevel: debug
-
+  signalfx/internal:
+    access_token: "${SPLUNK_ACCESS_TOKEN}"
+    realm: "${SPLUNK_REALM}"
+    sync_host_metadata: true
```

Comment:
Can you please move it up a bit closer to the other `signalfx` exporter?

Reply:
done